### DISCUSSION TITLE
Somthing wrong with 02_fwi_example_NLopt

### DISCUSSION THREAD

**Discussion by venividivici666** [slimgroup/JUDI.jl#191] [Q&A]
![image](https://github.com/slimgroup/JUDI.jl/assets/60508154/1b95a331-bebd-45ce-9061-23702a6e1167)

**Comment by mloubout** [comment]
Can I ask what versions of JUDI and Optim you are using? These examples are all run as part of the CI so they should work

**Reply by venividivici666** [reply]
I should be using the latest versions of JUDI and Optim

**Reply by mloubout** [reply]
`@time (minf, minx, ret) = optimize(opt, model0.m[:])` should fix it

**Reply by venividivici666** [reply]
Thanks a lot
